onnx/defs/math/defs.cc

```diff
@@ -1046,4 +1046,28 @@ ONNX_OPERATOR_SET_SCHEMA(
             "Constrain input and output types to float tensors.")
         .TypeAndShapeInferenceFunction(propagateShapeAndTypeFromFirstInput));
 
+static const char* Expand_ver8_doc = R"DOC(
+Expand the input tensor following the given shape.
+)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+	Expand,
+	8,
```

Comment:
Need to bump the opset here https://github.com/onnx/onnx/blob/033f956f41c55fd409e1c4a0d09795ae5411447f/onnx/defs/schema.h#L588, since 7 has been frozen in release 1.2 already.